src/remote.sh

```diff
@@ -257,3 +258,63 @@ function get_remote_info()
   echo "$ip"
   return 0
 }
+
+# This function manages ssh operations. Currently, we have three basic actions:
+# connect to the VM, execute a command in the VM, and perform a local script
+# inside the VM.
+#
+# @opts Expects a parameter to be passed for the ssh command (it could be
+#       --command, -c, --script, -s). If this parameter receives a null value
+#       this function will make a simple ssh operation; otherwise, it will
+#       attempt to execute a command or script.
+function vm_ssh
+{
+  local opts=$@
+  local port=${configurations[ssh_port]};
+  local target=${configurations[ssh_ip]};
+
+  if [[ "$1" == -h ]]; then
+    ssh_help
+    exit 0
+  fi
+
+  # Mandatory parameter
+  if [ -z "$target" ]; then
+    complain "Invalid argument: $@"
+    complain "Take a look at the config file, something is wrong in the ssh_ip"
+    exit 22 # EINVAL
+  fi
+
+  if [[ $# -gt 0 ]]; then
```

Comment:
`/$#/"$#"/`

Reply:
I don't think I understand what you meant with this.
Do you want me to consider double quoting the $# ?

Reply:
Replace `$#` for `"$#"`.